media-source/mediasource-config-changes.js

```diff
@@ -64,20 +64,7 @@ function mediaSourceConfigChangeTest(directory, idA, idB, description)
                 test.waitForExpectedEvents(function()
                 {
                     assert_false(sourceBuffer.updating, "updating");
-                    assert_greater_than(mediaSource.duration, 2, "duration");
-
-                    // Truncate the presentation to a duration of 2 seconds.
-                    mediaSource.duration = 2;
-
-                    assert_true(sourceBuffer.updating, "updating");
-                    test.expectEvent(sourceBuffer, 'updatestart', 'sourceBuffer');
-                    test.expectEvent(sourceBuffer, 'update', 'sourceBuffer');
-                    test.expectEvent(sourceBuffer, 'updateend', 'sourceBuffer');
-                });
-
-                test.waitForExpectedEvents(function()
-                {
-                    assert_false(sourceBuffer.updating, "updating");
+                    assert_less_than(mediaSource.duration, 5, "duration");
```

Comment:
Current mediasource-config-changes.js uses explicit remove(2,...) and is more complete. This part of the this change is done better by current test.